packages/smart-contracts/package.json

```diff
@@ -47,6 +47,7 @@
     "lint:check": "yarn run lint:lib:check && yarn run lint:sol:check",
     "ganache": "ganache-cli -l 90000000 -p 8545 -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\"",
     "deploy": "yarn hardhat deploy-local-env --network private",
+    "deploy-batch-payments": "yarn hardhat deploy-batch-payments --network rinkeby",
```

Comment:
That's right, I can delete it.
-> I added it because I used it a lot to test on rinkeby, and it would be useful when I will work again on the contract, e.g: add batch conversion. And also when we'll update contracts for gas optim.